themes/digital.gov/layouts/partials/core/collection.html

```diff
@@ -90,7 +90,7 @@ <h2>{{- .heading -}}</h2>
   {{- $src := .src -}}
   {{- with .item -}}
     <div class="icon">
-      <a href="{{- $href -}}" title="{{- .Title | markdownify -}}">
+      <a href="{{- $href -}}" title="{{- .Title | markdownify -}}" aria-hidden="true">
         <img src="{{ $src | relURL }}" alt="{{- .Params.name }} icon"/>
       </a>
     </div>
```

Comment:
Is this used in other places? I was wondering why it was a link on it's own. We could simplify it if it's decorative.

```diff
-    <div class="icon">
-      <a href="{{- $href -}}" title="{{- .Title | markdownify -}}" aria-hidden="true">
-        <img src="{{ $src | relURL }}" alt="{{- .Params.name }} icon"/>
-     </a>
-    </div>
+  <img src="{{ $src | relURL }}"  alt="" aria-hidden="true" />
```

Reply:
Yes, this partial is used in:

- `layouts/home.html`
- `layouts/resources/list.html`
- `layouts/topics/list.html`

We'll still need the `div` wrapper to maintain proper sizing.
I've removed the link wrapper.